R/test-elements.R

```diff
@@ -989,6 +1003,15 @@ volume_calibration_page <- function(url, type = tools::file_ext(url),
                   wait = FALSE, loop = TRUE,
                   admin_ui = admin_ui,
                   btn_play_prompt = btn_play_prompt)
+  audio_NAFC_page(label = "volume_calibration",
```

Comment:
Maybe delete the duplicated `audio_NAFC_page` call above that is currently being ignored?